### PR TITLE
add jsk_fetch package

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

File: jsk_fetch_robot/fetcheus/package.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <name>fetcheus</name>
+  <version>1.0.1</version>
+  <description>The naoeus package</description>
```
Comment (furushchev) [inline jsk_fetch_robot/fetcheus/package.xml]: this should be fetcheus?

**Comment by k-okada** [comment]
DO NOT MERGE YET

**Comment by k-okada** [comment]
参考。https://gist.github.com/k-okada/548bd323837e2504fb1a

**Comment by k-okada** [comment]
rewrited in #561

**Pull request closed without merging**